docs/source/dev_guide/algorithms/kmeans/k-means-clustering.rst

```diff
@@ -221,33 +222,28 @@ Initialization
 **************
 
 The K-Means clustering algorithm requires initialization of
-centroids as an explicit step. Initialization flow depends by the
+centroids as an explicit step. Initialization flow depends on the
 computation mode. Skip this step if you already calculated initial
 centroids.
 
-Batch Processing
-----------------
+For initialization, the following computation modes are available:
 
-.. include:: includes/initialization-batch.rst
-
-Distributed Processing
-----------------------
-
-.. include:: includes/initialization-distributed.rst
+.. toctree::
+   :maxdepth: 1
+   
+   initialization-batch.rst
+   initialization-distributed.rst
 
 Computation
 ***********
 
+The following computation modes are available:
```

Comment:
I propose to clarify here why there are 2 computation modes and only batch examples are exists in next section. In previous version it was 'At this moment, the description of distributed processing for K-Means computation is only available in Developer Guide for Intel(R) DAAL. Examples:'. It was not ideal, but it can be considered as 'Distributed in DAAL, DAAL is C++ and Java, not DPC++'. But in your new version, it becomes totally unclear, why after mentioning of 2 computation modes, there is no distributed example for DPC++.

Reply:
Can it be a note after that line (and right above the examples section) "Note: Distributed processing is not available in DPC++"?

Reply:
Can we mention tech preview distributed Kmeans for DPC++ ?

Reply:
"Note: Distributed processing in DPC++ is only available as a technical preview feature. Refer to [link to readme] for details."?

Reply:
@averbukh  I've added a note, see line 248